src/test/ui/unsafe/rfc-2585-unsafe_op_in_unsafe_fn.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(unsafe_block_in_unsafe_fn)]
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Please make this `warn`, to make sure that is sufficient to trigger the "unused_unsafe" lint.